### PR TITLE
Sprint 2: user profile 

### PR DESCRIPTION
**Pull request by NureHorishniaKateryna**
- added my profile page with new UI
- added reset passsword page

### CODE REVIEW

File: my-app/src/app/pages/my-profile/my-profile.component.html [hunk 1]
```diff
@@ -51,30 +51,28 @@ <h2>My Profile</h2>
...
+        </li>
+      </ul>
     </div>
+
     <div class="reservations">
       <div class="reservations-filter">
         <h2>My reservations</h2>
       </div>
 
       <div *ngFor="let ticket of tickets" class="reservation">
```
Comment (ArRuslan) [inline my-app/src/app/pages/my-profile/my-profile.component.html:73]: rename ticket to reservation

Comment (ArRuslan) [inline my-app/src/app/pages/my-profile/my-profile.component.html]: rename ticket to reservation

File: my-app/src/app/pages/my-profile/my-profile.component.ts [hunk 2]
```diff
@@ -17,10 +17,17 @@ export class MyProfileComponent implements OnInit {
   userData: any; // Змінна для збереження інформації про користувача
   isEditing: boolean = false;
   avatarUrl: string | undefined;
-  tickets: TicketData[] = [];
```
Comment (ArRuslan) [inline my-app/src/app/pages/my-profile/my-profile.component.ts:20]: rename tickets to reservations

File: my-app/src/app/pages/my-profile/my-profile.component.ts [hunk 3]
```diff
@@ -204,7 +211,7 @@ export class MyProfileComponent implements OnInit {
     }
   }
 
-  showTicketInfo(ticket: TicketData) {
+  showTicketInfo(ticket: any) {
```
Comment (ArRuslan) [inline my-app/src/app/pages/my-profile/my-profile.component.ts:202]: why any?

**Review by ArRuslan — CHANGES_REQUESTED** [review]

File: my-app/src/app/pages/my-profile/my-profile.component.html (hunk 1 above)
Comment (NureHorishniaKateryna) [reply my-app/src/app/pages/my-profile/my-profile.component.html]: all variables names will be renamed later when API will be ready to implement in on this page. For now we use just mock data to show reservations on my profile page so this component just in development stage.I will do refactor after implemented API

**Review by ArRuslan — APPROVED** [review]

File: my-app/src/app/services/open-api.service.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ export class OpenApiService {
     });
     console.log(localStorage.getItem('token'));
 
-    return this.http.get<any>(`${this.apiUrl}/users/me`, { headers: headers });
+    return this.http.get<any>(`${this.apiUrl}/user/info`, { headers: headers });
```
Comment (ArRuslan) [inline my-app/src/app/services/open-api.service.ts:27]: also, change updateUserProfile to this endpoint

**Review by ArRuslan — CHANGES_REQUESTED** [review]
we dont need user password when editing profile

File: my-app/src/app/services/open-api.service.ts (hunk 4 above)
Comment (NureHorishniaKateryna) [reply my-app/src/app/services/open-api.service.ts:27]: fixed

Comment (ArRuslan) [reply my-app/src/app/services/open-api.service.ts:27]: no it's not

**Review by ArRuslan — APPROVED** [review]

**Pull request merged**